src/cryptoadvance/specter/util/fee_estimation.py

```diff
@@ -69,5 +146,13 @@ def get_fees(specter, config):
         "minimumFee": int(
             (float(specter.estimatesmartfee(20).get("feerate", 0.00001)) / 1000) * 1e8
         ),
-        "failed": "feerate" not in specter.estimatesmartfee(1),
     }
+    if "feerate" not in specter.estimatesmartfee(1):
+        # regtest does not seem to have a reasonable result for estimatesmartfee.
+        # The error-message is covering "transactions" and so the cypress-tests are not succeeding
+        # Not a perfect fix but better than to fix it everywhere in the test-code:
+        if specter.chain != "regtest":
+            fee_estimation_result.error_message = "There was an issue while fetching fee estimation with  Bitcoin core. Please use manual fee estimation"
```

Comment:
Maybe we could also append here error message from Bitcoin Core?
The result from failed `estimatesmartfee` looks like this:
```
{
  "errors": [
    "Insufficient data or no feerate found"
  ],
  "blocks": 0
}
```